### PR TITLE
Nuke: Publishing, loading and updating alembic cameras

### LINKED ISSUE #565: Nuke Camera Loader
<div> Freshdesk Ticket ID: <a href="https://pype.freshdesk.com/a/tickets/255">255</a></div>
                       <div> Freshdesk Ticket Agent: Jakub Jezek</div>
                       <div> Ticket Priority: Medium</div>
                       <div> Freshdesk Ticket Description: </div><div style="font-family:-apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Arial, sans-serif; font-size:14px"><div dir="ltr">Can we please add camera loader to nuke?</div></div>

### PR DESCRIPTION
**Pull request by jezscha**
- PR #566  already merged into the branch for testing purposes
- publishing internal camera node with baked animation from parent axis's. Baked on word coordinate matrix XYZ so it should be working well even with Maya.

Close #565

### CODE REVIEW

**Review by mkolar — CHANGES_REQUESTED** [review]
Publishing and loading is working fine with one strange effect. When loading animated ABC to nuke. it doesn't show animation, until I manually disable and enable reading from the file. then it catches on correctly. Can you please have a look at that?

Updating and removing needs to be added as well, before merging.

File: pype/plugins/nuke/load/load_camera_abc.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        file = self.fname.replace("\\", "/")
+
+        camera_node = nuke.createNode(
+            "Camera2",
+            "file {} read_from_file True".format(file),
+            inpanel=False
+        )
+        camera_node.forceValidate()
+        # camera_node["read_from_file"].setValue(True)
+        # camera_node["file"].setValue(file)
```
Comment (mkolar) [inline pype/plugins/nuke/load/load_camera_abc.py:54]: if these lines are not needed let's remove them.

**Comment by jezscha** [comment]
> Publishing and loading is working fine with one strange effect. When loading animated ABC to nuke. it doesn't show animation, until I manually disable and enable reading from the file. then it catches on correctly. Can you please have a look at that?
@mkolar I believe it is some internal Nuke bug as this is happening to me to. Didn't find any trace of evidence anybode had noticed it within community. When I had programmatically hit Reload button it want work either, although it is working in gui - try it.  
Not sure what to do with this really, @tokejepsen any idea here?

**Comment by tokejepsen** [comment]
Dont really know about this, but you could force the node to into the control panel if that fixes the issue.

**Pull request merged**